src/stylesheets/base/_layout.scss

```diff
@@ -0,0 +1,3 @@
+html {
+  font-size: 62.5%;
```

Comment:
why do we need this setting?

Reply:
It makes the `rem` conversion much easier and readable. [Have a look at this answer](https://stackoverflow.com/a/43554897/7903039)